packages/edit-post/src/components/header/style.scss

```diff
@@ -31,6 +31,55 @@
 	}
 }
 
+.is-distraction-free {
+	.interface-interface-skeleton__header {
+		border-bottom: none;
+	}
+
+	.edit-post-header {
+		position: absolute;
+		width: 100%;
+		background-color: rgba(255, 255, 255, 0.9);
+		backdrop-filter: blur(20px) !important;
+		border-bottom: 1px solid #e0e0e0;
+
+
+		// hide some parts
+		& > .edit-post-header__settings > .editor-post-preview {
+			visibility: hidden;
+		}
+
+		& > .edit-post-header__toolbar .edit-post-header-toolbar__inserter-toggle,
+		& > .edit-post-header__toolbar .edit-post-header-toolbar__list-view-toggle {
+			display: none;
+		}
+
+		& > .edit-post-header__settings > .block-editor-post-preview__dropdown {
+			display: none;
+		}
+
+		& > .edit-post-header__settings > .interface-pinned-items {
+			display: none;
```

Comment:
Can we combine these display none?

Reply:
Fixed in [e745879](https://github.com/WordPress/gutenberg/pull/41740/commits/e74587928f829a76db497d29df4308fa2343939a)